build/edge/docker-compose.yaml

```diff
@@ -5,8 +5,18 @@ networks:
     driver: bridge
 
 services:
+  dind:
+    image: docker:dind
+    restart: always
+    privileged: true 
+    networks:
+      - kubeedge-bridge
   emqx:
-    image: emqx/emqx-edge:v3.0.1-${ARCH}
+    image: emqx/emqx-edge:v3.1.0-${ARCH}
+    environment:
+      EMQX_STORM__ADDRESS: ${EMQX_STORM_ADDRESS}
```

Comment:
I think we have multiple "_" after STORM ...

Reply:
emqx will modify the configuration item through the environment variable, the two ‘_’ are the rules it sets, so this is correct

Reply:
Sounds good :)